extensions/index.md

```diff
@@ -8,75 +8,6 @@ title: Overview
 
 One of the primary goals of Reactor is to create an open ecosystem where engineers outside of the core Launch engineering team can expose additional functionality through Launch. This is done through Reactor extensions. Once an extension has been installed on a Launch property by a user, that extension's functionality then becomes available for use by all users of the property.
 
-The primary components of an extension include the following:
+To find out more about web extensions, you can go [here](web_extensions).
```

Comment:
Take it or leave it, but I would probably do `[Find out more about web extensions.](web_extensions)`